extensions/3DTILES_extent/README.md

```diff
@@ -0,0 +1,136 @@
+
+## Defining Extents
+
+An extent is a collection of latitude and longitude coordinate pairs. Extents are two dimensional in nature, but an optional third component can be specified for **each** extent coordinate to specify its height (in meters). The coordinate pairs should be provided in **counterclockwise** winding order. Multiple extents can be specified. Convex and concave extents are both supported. At least three coordinates must be provided for an extent to be valid.
+
+The extent region definitions can be directly embedded in the `tileset.json` or located in a separate file and referred to using a `uri` reference in the corresponding `tileset.json` file.
+
+### External extent definition
+
+`tileset.json`
+
+```json
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "extensionsUsed": [
+    "3DTILES_extent"
+  ],
+  "extensions": {
+    "3DTILES_extent": {
+      "uri": "extent.json"
+    }
+  },
+  "geometricError": 100.0,
+  "root": {...}
+}
+```
+
+### `extent.json`
+
+```json
+{
+  "polygons": [
+    {
+      "coordinates": [
+        [32.511166344475825, -116.61512629247555],
+        [32.514456825587125, -116.61385130521423],
+        [32.515638616140173, -116.61829305963617],
+        [32.513154268152434, -116.61944030086889],
+        [32.511166344475825, -116.61512629247555]
+      ]
+    }
+  ]
+}
+```
+
+### Embedded extent definition
+
+`tileset.json`
+
+```json
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "extensionsUsed": [
+    "3DTILES_extent"
+  ],
+  "extensions": {
+    "3DTILES_extent": {
+      "extent": {
+        "polygons": [
+          {
+            "coordinates": [
+              [32.511166344475825, -116.61512629247555],
+              [32.514456825587125, -116.61385130521423],
+              [32.515638616140173, -116.61829305963617],
+              [32.513154268152434, -116.61944030086889],
+              [32.511166344475825, -116.61512629247555]
+            ]
+          }
+        ]
+      }
+    }
+  },
+  "geometricError": 100.0,
+  "root": {...}
+}
+```
+
+## Holes in Extents
+
+Holes are also supported, simpliy provide a `holes` object for a given polygon, e.g:
+
+```json
+  "extensions": {
+    "3DTILES_extent": {
+      "extent": {
+        "polygons": [
+          {
+            "coordinates": [
+              [32.511166344475825, -116.61512629247555],
+              [32.514456825587125, -116.61385130521423],
+              [32.515638616140173, -116.61829305963617],
+              [32.513154268152434, -116.61944030086889],
+              [32.511166344475825, -116.61512629247555]
+            ],
+            "holes": [
+                [32.2032301338380, -116.61521629247]
+                [32.3482030238302, -116.323208320832]
+                [32.7382033280322, -116.4830243804380]
+            ]

```

Comment:
I think this would only let you defined a single hole.

Reply:
It's a 3D array now.